AutoCollection/ServerRequestParser.ts

```diff
@@ -34,8 +34,18 @@ class ServerRequestParser extends RequestParser {
             this.rawHeaders = request.headers || (<any>request).rawHeaders;
             this.socketRemoteAddress = (<any>request).socket && (<any>request).socket.remoteAddress;
             this.userAgent = request.headers && request.headers["user-agent"];
-            this.sourceIKeyHash =
-                request.headers && request.headers[RequestResponseHeaders.sourceInstrumentationKeyHeader];
+
+            if (request.headers && request.headers[RequestResponseHeaders.requestContextHeader]) {
+                const keyValues = request.headers[RequestResponseHeaders.requestContextHeader].split(",");
```

Comment:
Consider moving this header parsing logic to `Utils` and reuse it in both `ClientRequestParser`  and `ServerRequestParser`.